src/extension.ts

```diff
@@ -44,6 +44,16 @@ export function activate(context: vscode.ExtensionContext) {
             node.openInPortal();
         }
     });
+    initAsyncCommand(context, 'subscription.CreateWebApp', async (node: SubscriptionNode) => {
+        if (node && node.openInPortal) {
+            const wizard = new WebAppCreator(outputChannel, azureAccount, node.subscription);
+            const result = await wizard.run();
+            
+            if (result.status === 'Completed') {
+                vscode.commands.executeCommand('appService.Refresh', node);
```

Comment:
I know the code was like this before your PR, but since you're in the same area, can you change it to `appServiceDataProvider.refresh(node)`?

I don't like the fact that `executeCommand` relies on a hard-coded string (`appService.Refresh`). If we call `appServiceDataProvider.refresh` directly, we get intellisense, automatic renaming, etc.